src/main/java/uk/gov/ons/census/casesvc/model/dto/FulfilmentRequestDTO.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.ons.census.casesvc.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class FulfilmentRequestDTO {
+
+  @JsonInclude(Include.NON_NULL)
+  @JsonProperty("caseId")
```

Comment:
don't need this (appreciate it's in receipt & refusal which is redundant there too)